docs/GettingStarted.md

```diff
@@ -838,6 +838,7 @@ connection.get('/foo')
 | `distributed_tracing` |                                 | Enables [distributed tracing](#distributed-tracing)                                                                                                                                         | `true`    |
 | `split_by_domain`     |                                 | Uses the request domain as the service name when set to `true`.                                                                                                                             | `false`   |
 | `error_handler`       |                                 | A `Proc` that accepts a `response` parameter. If it evaluates to a *truthy* value, the trace span is marked as an error. By default only sets 5XX responses as errors.                      | `nil`     |
+| `on_error`      | `Proc`                  | While `error_handler` is a way to control how errors are set based on the response to a request, `on_error` is applied to other types of `Faraday` errors. For example, if the request fails because the server is not available, `on_error` will e called instead of `error_handler`. This configuration overrides error handling behavior when a span raises an error. Provided `span` and `error` as arguments. Sets error on the span by default. | `proc { |span, error| span.set_error(error) unless span.nil? }` |
```

Comment:
```suggestion
| `on_error`      | `Proc`                  | While `error_handler` is a way to control how errors are set based on the response to a request, `on_error` is applied to other types of `Faraday` errors. For example, if the request fails because the server is not available, `on_error` will be called instead of `error_handler`. This configuration overrides error handling behavior when a span raises an error. Provided `span` and `error` as arguments. Sets error on the span by default. | `proc { |span, error| span.set_error(error) unless span.nil? }` |
```